src/DFAMin.hs

```diff
@@ -25,7 +24,7 @@ import qualified Data.List as List
 --      choose and remove a set A from Q
 --      for each c in ∑ do
 --           let X be the set of states for which a transition on c leads to a state in A
```

Comment:
Can we rename `X` to `Preimage` here and below?

Reply:
I'll take this one back, then w gotta rename a bunch of other things, and the `refines` already adds clarity.